packages/hardhat-etherscan/src/types.ts

```diff
@@ -1,43 +1,40 @@
-/* eslint-disable @typescript-eslint/naming-convention */
-
-export enum Chains {
-  mainnet = "mainnet",
-  ropsten = "ropsten",
-  rinkeby = "rinkeby",
-  goerli = "goerli",
-  kovan = "kovan",
+type Chain =
```

Comment:
Converted this to a union type instead of an enum. I think this simplifies things a bit, but there might be an issue with this approach that I'm not seeing. Please let me know if that's the case and I'll revert this change.

Reply:
I don't know any reason, other than a hazy preference for enums. Do we have a general preference for string unions over enums?

Reply:
It just feels slightly better in this case. It's a bit less code and IMO it's closer to the intention of what we want here (a list of keys). In other words, the way the enum was used was equivalent to just doing this, and without any benefit from using an enum.

The other reason that comes to mind is that enums exist at runtime, which makes them a more hybrid/complex thing. Union types are just types, so thinking about them is easier (again, IMO).